config/crd/bases/infrastructure.cluster.x-k8s.io_firewallrules.yaml

```diff
@@ -0,0 +1,143 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.16.5
+  labels:
+    clusterctl.cluster.x-k8s.io/move-hierarchy: "true"
+  name: firewallrules.infrastructure.cluster.x-k8s.io
```

Comment:
nit: Would calling this CRD `LinodeFirewallRule` be better since we're using the shared `infrastructure.cluster.x-k8s.io` group here and `github.com/linode/linodego`-specific types inside the definition?

Reply:
I didn't want to do that because I want Linode* resources to correspond to a cloud resource. For example, AddressSet does not.